qualtran/_infra/gate_with_registers.py

```diff
@@ -172,17 +175,21 @@ def add_my_tensors(
         incoming: Dict[str, 'SoquetT'],
         outgoing: Dict[str, 'SoquetT'],
     ):
-        from qualtran.cirq_interop._cirq_to_bloq import _add_my_tensors_from_gate
-
-        _add_my_tensors_from_gate(
-            self,
-            self.signature,
-            self.short_name(),
-            tn=tn,
-            tag=tag,
-            incoming=incoming,
-            outgoing=outgoing,
-        )
+        try:
+            # First try to add tensors via decomposition.
+            Bloq.add_my_tensors(self, tn, tag, incoming=incoming, outgoing=outgoing)
+        except DecomposeNotImplementedError:
+            from qualtran.cirq_interop._cirq_to_bloq import _add_my_tensors_from_gate
+
+            _add_my_tensors_from_gate(
```

Comment:
I'm just ... can you document the use case of when we'd rely on this fallback vs the other? You know I don't like multiple default fallbacks. 

The alternative would be for bloq(/gate) authors to be explicit about which fallback they're relying on.

Reply:
not relevant anymore. See comment above.